modules/user/user.service.js

```diff
@@ -105,6 +105,45 @@ class UserService {
     );
   }
 
+  async loginAdmin({ email, password }) {
+    const { errors } = await validateLoginInput.validateAsync({
+      email,
+      password,
+    });
+
+    if (errors) {
+      throw new UserInputError('Errors', {
+        errors,
+      });
+    }
+
+    const user = await this.getUserByFieldOrThrow('email', email);
+
+    const match = await bcrypt.compare(
+      password,
+      user.credentials.find(cred => cred.source === 'horondi').tokenPass,
+    );
+
+    if (user.role === 'user') {
+      throw new AuthenticationError(`No access!`);
```

Comment:
Please, use the uppercased error message as it is done for the news, for example

Reply:
Changed!